src/styles/chat.css

```diff
@@ -0,0 +1,327 @@
+.logo {
```

Comment:
good to have customized css. Just suggestive note: we could use bootstrap or other framework to easy up our styling job.